packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -258,10 +265,18 @@ export default function isConformant(
       });
 
       test('passes extra props to the component it is renders as', () => {
-        const MyComponent = () => null;
-        const wrapper = mount(<Component {...requiredProps} as={MyComponent} data-extra-prop="foo" />);
-
-        expect(wrapper.find('MyComponent[data-extra-prop="foo"]').length).toBeGreaterThan(0);
+        if (passesUnhandledPropsTo) {
+          const wrapper = mount(<Component {...requiredProps} data-extra-prop="foo" />).find(
+            passesUnhandledPropsTo.displayName,
+          );
+          expect(wrapper.find(`${passesUnhandledPropsTo.displayName}[data-extra-prop="foo"]`).length).toBeGreaterThan(
+            0,
+          );
+        } else {
+          const MyComponent = () => null;
+          const wrapper = mount(<Component {...requiredProps} as={MyComponent} data-extra-prop="foo" />);
+          expect(wrapper.find('MyComponent[data-extra-prop="foo"]').length).toBeGreaterThan(0);
```

Comment:
```suggestion
          expect(el.prop('data-extra-prop')).toBe('foo');
```

I would simplify this and make it more strict